### PR TITLE
Add wireframe, DOM manipulation and App logic.

### PR DESCRIPTION
**Pull request by sergiogval**
# Awesome books 

### This PR includes the following requirements for the app project 1: 

- [x]   Create a collection that keeps a list of books.

- [x]     Create a function to add a new book to the collection, with title and author.

- [x]     Create a function to remove a book from the collection.

- [x]     Display all books saved in the collection in the top part of the page.

- [x]     Make sure that when a user clicks the "Add" button:
        - A new book is added to the collection.
        - The new book is displayed in the page.
        
- [x] Make sure that when a user clicks the "Remove" button:
        - The correct book is removed from the collection.
        - The correct book dissapears from the page.
        
- [x]   Make sure that data is preserved in the browser's memory by using localStorage.

- [x] Don't use any library or framework, use only plain JavaScript.

- [x]   You need to use pair-programming for this project. We will check to see if both partners submitted code by looking at the commit history for each project. Therefore you need to use only one repository for your pair-programming group.

### CODE REVIEW

File: logic.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+  e.preventDefault();
+  // eslint-disable-next-line no-use-before-define
+  addBooks();
+  // eslint-disable-next-line no-use-before-define
+  remove();
+  // eslint-disable-next-line no-use-before-define
+  saveLocalStore();
+});
+
+function addBooks() {
```
Comment (rloterh) [inline logic.js:42]: - [x] Entries are **not able to be persisted successfully** to the _local storage_. When the _Add book_ button is clicked after entering some data, and subsequently, the page is refreshed the entered data vanishes from the local storage. Local storage shows blank entries. Kindly improve upon this aspect.

File: logic.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+  }, {
+    title: 'The Most Human Human',
+    author: 'Brian Christian',
+  }, {
+    title: '1984',
+    author: 'George Orwell',
+  }];
+
+const form = document.getElementById('form');
+const title = document.getElementById('title');
```
Comment (rloterh) [inline logic.js:14]: - [x] An error is traced in your _Logic.j_s file. Kindly endeavor to fix this issue.

**Error:**
image

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Shay, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦 🚦 🚦  

## Good Points 👍  
- Descriptive Pull Request
- No linter errors


## Required Changes ♻️
_Kindly check the comments under the review._

<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: logic.js (hunk 2 above)
Comment (sergiogval) [reply logic.js:14]: About this suggestion, I can´t find the requirement asking for no error on JS console. That said, I believe this error will be solved by if the localStore bug. If that is not the case, I would kindly ask for a second opinion about this.

Comment (rloterh) [reply logic.js:14]: Hi @sergiogval 
You are right! This request is supposed to be optional.
I apologize for omitting this important step during my review. Kindly note that this was not a deliberate attempt!

Cheers and happy coding! 😃

**Review by mikethreels — APPROVED** [review]
Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 :
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**